gradle-plugin/README.md

```diff
@@ -51,48 +86,10 @@ repository into Android Studio.
 
 ### Developing the plugin against local repositories
 Like [the android components suggest][components local], it's preferable to avoid depending on apps outside of the repository. Instead:
-- Write unit tests
-- Work against the sample app (TODO: add sample app :)
-
-If you wish to develop against local repositories anyway, see below.
-
-#### Set a version number
-Append `-SNAPSHOT` to the end of the version number in this project's build.gradle.kts. If you don't, your imports may conflict with those from remote repositories and you can end up polluting your local maven cache.
-
-#### Publish to your local maven repository
-To publish the gradle plugin:
-```sh
-./gradlew gradle-plugin:publishToMavenLocal
-```
-
-#### Import your local plugin changes into the local repository
-In the top level gradle file, you have to add the dependencies (using your own version number):
-```groovy
-buildscript {
-    repositories {
-        mavenLocal()
-    }
-
-    dependencies {
-        classpath "org.mozilla.android:gradle-plugin:0.1-SNAPSHOT"
-    }
-}
-
-apply plugin: org.mozilla.android.MozillaPlugin
-```
-
-And again in `buildSrc/build.gradle`:
-```groovy
-repositories {
-    mavenLocal()
-}
-
-dependencies {
-    implementation("org.mozilla.android:gradle-plugin:0.1-SNAPSHOT")
-}
-```
+- Write unit tests and/or Gradle TestKit tests.
+- Work against the sample app in `samples/app`.  For example, to test consuming a local GeckoView, modify `samples/local.properties` and run `./gradlew -p samples app:assembleDebug`, etc.
 
-If your project applies the plugin from a remote maven repository, be sure to remove that application.
+If you wish to develop against other local repositories, it's best to use Gradle composite builds, following the model of the samples.
```

Comment:
Which samples demonstrate this? Are we actually using composite builds (e.g. for GV or the gradle-plugin substitution)? I was unaware and I think it could be confusing to others to hear this out of context: maybe we can add a note earlier, e.g. "You can use a local GV via Gradle composite builds: just follow the simple instructions". That way, when devs see, "it's best to use Gradle composite builds", they know they're already doing this.